Commons/src/main/java/eu/neclab/ngsildbroker/commons/datatypes/QueryTerm.java

```diff
@@ -776,6 +776,10 @@ private void getAttribQueryV2(StringBuilder result) throws ResponseException {
 			}
 			applyOperator(attributeFilterProperty);
 			attributeFilterProperty.append(" OR ");
+			attributeFilterProperty.append('(');
+			attributeFilterProperty.append(operant.replaceAll("\"","\'"));
+			attributeFilterProperty.append(" in (select jsonb_array_elements("+charcount+"->'https://uri.etsi.org/ngsi-ld/hasValue')->>'@value'))");
```

Comment:
[Minor] please change the strings in general to the matching constants as "hasValue" and' @value'.